lib/charms/microcluster_token_distributor/v0/token_distributor.py

```diff
@@ -44,6 +44,32 @@ def get_hostname():
     return os.uname().nodename
 
 
+def corroborate(items: list, default=""):
+    """Return the most frequent non default value in a list.
```

Comment:
This docstring makes it sound that the function won't return `default` value, but it does in multiple cases.